app/assets/javascripts/discourse/config/deprecation-workflow.js

```diff
@@ -9,5 +9,6 @@ globalThis.deprecationWorkflow.config = {
     { handler: "silence", matchId: "this-property-fallback" },
     { handler: "silence", matchId: "discourse.select-kit" },
     { handler: "silence", matchId: "discourse.modal-controllers" },
+    { handler: "silence", matchId: "discourse.inline-date" },
```

Comment:
Is this actually being hit in core? Would be good to resolve it, or at least have a plan/timeline.

Reply:
It's used in badges/show template. Should be small task, hopefully I'll be able to do it between other things.